src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -1918,12 +1924,21 @@ namespace NServiceBus.Performance.TimeToBeReceived
         public System.TimeSpan MaxTime { get; }
     }
 }
-namespace NServiceBus.Persistence.Legacy
```

Comment:
Strange. I've removed it as part of another PR but I'll bring it back to develop ASAP (related to https://github.com/Particular/PlatformDevelopment/issues/133)